buildbot-host/buildmaster/openvpn/check_steps.cfg

```diff
@@ -0,0 +1,31 @@
+# -*- python -*-
+# ex: set filetype=python:
+def openvpnAddCheckStepsToBuildFactory(factory, combo, shell_env=None):
+    if shell_env is None:
+        shell_env = {}
+
+    # Run the tests, including t_client.sh and t_server_null.sh
+    t_client_env = {"TCLIENT_SKIP_RC": "1"}
```

Comment:
Note that this will break `make check` if t_client is not enabled. This ensures it can't be silently skipped.

Reply:
Hmm, yes, "make  check" assumes all is good if exit code of t_client.sh is 77 (i.e. some dependency is missing). I'm working on a fix.

Reply:
I pushed a fix that seems to do the trick. Basically, if *t_client* tests are enabled in *master.ini* then *TCLIENT_SKIP_RC=1* is added to the shell environment.